examples/02-plot/moving_cmap.py

```diff
@@ -0,0 +1,81 @@
+"""
+.. _moving_cmap_example:
+
+Create a GIF Movie of a Static Object with a Moving Colormap
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Generate a gif movie of a Hopf torus with a moving colormap, 
+by updating the scalars.
+
+"""
+
+from math import pi
+
+import numpy as np
+
+import pyvista as pv
+
+# A spherica curve

```

Comment:
```suggestion
# A spherical curve

```

Reply:
Ok, I do the changes. Me too, I like the result with the `nipy_spectral` colormap.